src/Castle.Core/DynamicProxy/AllMethodsHook.cs

```diff
@@ -32,6 +32,8 @@ public class AllMethodsHook : IProxyGenerationHook
 #endif
 		};
 
+		protected static readonly ICollection<Type> SkippedTypes = skippedTypes;
```

Comment:
It would have been simpler to just change the access modifier of `SkippedTypes` from `protected` to `protected internal`, however that would (somewhat surprisingly) have led to changes in the `ref/` contract files, which I'd like to avoid whenever possible.

Reply:
This is no longer necessary as we really don't need to access `SkippedTypes` at all (see your other review comment), so we don't need to change this code at all.